### PR TITLE
Updated Docs

### PR DESCRIPTION
**Pull request by Yashvendra**
Updated Docs #11436

### CODE REVIEW

File: documentation/modules/auxiliary/scanner/http/http_put.md [hunk 1]
```diff
@@ -15,30 +15,26 @@ This module can abuse misconfigured web servers to upload and delete web content
...
+Set `ACTION` to either `PUT` or `DELETE`. (Default: `PUT`)
+
 **PUT**
 
 Action is set to PUT to upload files to the server. If `FILENAME` isn't specified, the module will generate a random string as a .txt file.
 
 **DELETE** 
 
-Deletes the file specified in the `FILENAME` option. (default: `msf_http_put_test.txt`)
+`FILENAME` is required when Action is set to DELETE
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/http/http_put.md]: i think this line and the line that was removed are both relevant here.  The default file name is still good, and the explanation was solid.

File: documentation/modules/auxiliary/scanner/http/http_put.md [hunk 2]
```diff
@@ -15,30 +15,26 @@ This module can abuse misconfigured web servers to upload and delete web content
...
 **DELETE** 
 
-Deletes the file specified in the `FILENAME` option. (default: `msf_http_put_test.txt`)
+`FILENAME` is required when Action is set to DELETE 
 
 ### PATH
 
 The path at which this module will attempt to either PUT the content or DELETE it.
 
-### FILEDATA
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/http/http_put.md]: This should be added back, but the description is wrong.  The description should say something like 
```The content to put in the uploaded file if `action` is set to `PUT` ```.

File: documentation/modules/auxiliary/scanner/http/scraper.md [hunk 3]
```diff
@@ -7,15 +7,22 @@ This module scrapes data from a specific web page based on a regular expression.
...
 
+## Options
+
+### PATH 
+
+The path from where the data is to be scraped from.
+ 
+### PATTERN
+
+The regular expression to use . Default value:`<title>(.*)</title>` which simply grabs page title.
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/http/scraper.md]: grabs **the** page title

Comment (h00die) [reply documentation/modules/auxiliary/scanner/http/scraper.md]: A regular expression to capture data from the webpage.

Comment (Yashvendra) [reply documentation/modules/auxiliary/scanner/http/scraper.md]: Done.

**Review by h00die — APPROVED** [review]

**Comment by h00die** [comment]
# Release Notes
Aux http doc fixes

**Comment by h00die** [comment]
Thanks for the patience and updates on these!

**Pull request merged**